src/main/java/net/pms/network/HTTPResource.java

```diff
@@ -44,6 +44,7 @@
 	public static final String UNKNOWN_AUDIO_TYPEMIME = "audio/mpeg";
 	public static final String AUDIO_TRANSCODE = "audio/transcode";
 	public static final String VIDEO_TRANSCODE = "video/transcode";
+	public static final String AUDIO_AAC_TYPEMIME = "audio/mp4";
```

Comment:
> What it have to do with my work ?
> The fake AAC container set, for an easy reading, is only for MPEG-4 container (audio only)

ADTS have an ADTS container and MIME type. (.aac extension)
My work only concern .m4a (and incorrect file having an .mp4 extension), called fakely AAC, for an easy log reading. M4A ==> MPEG-4 for audio.

It's seem that we are not going to understand each other on that :-(